app/js/lib/foxbox.js

```diff
@@ -89,9 +89,19 @@ export default class Foxbox extends Service {
           this.togglePolling(settings.pollingEnabled);
         });
         this.togglePolling(settings.pollingEnabled);
+      }).then(() => {
+        window.foxbox = this;
       });
   }
 
+  /**
+   * Clear all data/settings stored on the browser. Use with caution.
+   */
+  clearAll() {
+    settings.clear();
+    return db.drop();
```

Comment:
When/If `settings.clear()` returns a promise, we can wrap it all in a call to `Promise.all` and return it directly.